templates/nginx/secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.expose.type "ingress" }}
```

Comment:
as for the nginx deployment, it also needs to be disabled in http-route scenario.

Reply:
as well as the cm of nginx.